benchutils/transformers.py

```diff
@@ -73,3 +73,23 @@ def last_entry(x):
         sub_df = all_ranks.loc[all_ranks['RANK'] == rank]
         sub_df_matching = sub_df[keep_cols]
         sub_df_matching.to_csv(output_, sep='\t', index=False)
+
+
+def mohawk_transformer(per_read_summary, output_summary):
```

Comment:
I will need some introduction to the "Mohawk rule"...

Reply:
[Mohawk](https://github.com/gwarmstrong/mohawk) is the deep learning based taxonomic classifier I have been working on. The `mohawk` CLI are coming from this module, and the transformer is implemented based on the form of the `mohawk` output. Currently, I only have a model that predicts at the genus level, so it is only implemented to predict genus.